CMakeLists.txt

```diff
@@ -344,7 +344,14 @@ endif()
 file(WRITE "${PROJECT_BINARY_DIR}/.hipVersion" ${_versionInfo})
 
 # Generate hipVersion.h
-set(_versionInfoHeader "//Auto-generated by cmake\n\n#define HIP_VERSION_MAJOR ${HIP_VERSION_MAJOR}\n#define HIP_VERSION_MINOR ${HIP_VERSION_MINOR}\n#define HIP_VERSION_PATCH ${HIP_VERSION_GITDATE}\n")
+set(_versionInfoHeader
+"// Auto-generated by cmake\n
+#ifndef HIP_VERSION_H
+#define HIP_VERSION_H\n
+#define HIP_VERSION_MAJOR ${HIP_VERSION_MAJOR}
+#define HIP_VERSION_MINOR ${HIP_VERSION_MINOR}
+#define HIP_VERSION_PATCH ${HIP_VERSION_GITDATE}\n
+#endif")
```

Comment:
And a new line at the end of the generated header file (after the closing bracket).